README.md

```diff
@@ -13,13 +15,24 @@ This is an [Outreachy](https://www.outreachy.org/) Internship project.
 This project uses [Phabricator](https://phabricator.wikimedia.org/project/board/6283/) to track issues and we would advice against using Github issue traking for bugs
 
 ## Setup/Installation Requirements
-* Clone this repo to your machine
-* `git clone https://github.com/wikimedia/toolhunt.git` 
-* `cd into toolhunt`
 
-## Technologies to be Used
-* Python
-* Flask
-* Redis
+- Clone this repo to your machine with the command `git clone https://github.com/wikimedia/toolhunt.git`
+- `cd toolhunt`
+
+### To run locally
 
+- Install Poetry and Flask if needed
+- `poetry run flask run`
+- Open a browser window to localhost:5000
+
+### To run with Docker
+
+- `docker-compose up`
```

Comment:
This runs the containers in the foreground. Adding the `--detach` option avoids this. Adding the `--build` option is also a good idea, in case the images are missing for some reason.